### DISCUSSION TITLE
Android: Lagspike cause game to speed up

### DISCUSSION THREAD

**Discussion by RQkaiju2** [ppy/osu#24815] [Q&A]
A certain Lagspike in osu!Lazer will make the game freeze/lag, after the lag it will speed up to where I'm im at in the beatmap, causing me to miss beats, break combos, and even lose the beatmap im playing in. If you can help me that would be amazing.

**Comment by peppy** [comment]
Performance and optimisation considerations are an ongoing effort. Especially on android devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations, or upgrade your device.
